src/azure-cli-core/HISTORY.rst

```diff
@@ -5,7 +5,7 @@ Release History
 
 2.0.29
 ++++++
-* Minor fixes
+* Allow flushing of events in telemetry and generic add-event capability
```

Comment:
Make this more user-facing or alternatively just leave as Minor fixes as it doesn't seem like there's a user-facing change here.